source/nsdynmemLIB/nsdynmemLIB.c

```diff
@@ -124,6 +126,25 @@ const mem_stat_t *ns_dyn_mem_get_mem_stat(void)
 #endif
 }
 
+int ns_dyn_mem_read_mem_stat(mem_gen_stat_t *mem_gen_stat)
+{
+#ifndef STANDARD_MALLOC
+    if (!default_book) {
+        return -1;
+    }
+
+    platform_enter_critical();
+    mem_gen_stat->heap_sector_size = default_book->heap_size;
```

Comment:
Does this count the memory off all separate books together?

Reply:
when you add more Sectors they are added to this size.